addons/mail/static/src/models/thread/thread.js

```diff
@@ -1995,6 +1996,12 @@ registerModel({
         guestMembers: many('Guest', {
             sort: '_sortGuestMembers',
         }),
+        /**
+         * States whether this thread has the read access.
```

Comment:
"States whether the current user has read access on this thread." would be a bit more clear I think